packages/maskbook/src/plugins/Wallet/services/wallet/password.ts

```diff
@@ -52,7 +54,7 @@ export function validatePassword(unverifiedPassword: string) {
 }
 
 export function validatePasswordRequired(unverifiedPassword: string) {
-    if (!validatePassword(unverifiedPassword)) throw new Error('An invalid password.')
+    if (!validatePassword(unverifiedPassword)) throw new Error('Incorrect payment password')
```

Comment:
This error means the password does not adhere to the basic validation rules. It hasn't check if the password is correct or not yet.